newsfragments/1571.misc.rst

```diff
@@ -0,0 +1 @@
+Add event reasons when an entry is updated for GUI.
```

Comment:
The newsfragment will end up in the changelog which is supposed to be read by users. So would be better to avoid internal details.

So I would rewrite ` newsfragments/1571.api.rst`:
>  Fix message name typing, setting it to EntryName to comfort with WorkspaceEntry. 
as:
> Fix incorrect definitions of entry name type for workspace and folder manifests in api

I consider advanced user are aware of workspace/folder manifests (see https://docs.parsec.cloud/en/latest/architecture.html#parsec-data-model), and the fact the API has been modified is important to know given it can create incompatibilities between versions

Considering `newsfragments/1571.misc.rst`, I realize it only concern internal changes for the moment so I think we should just remove this file. It will be the job of the PR using those changes to improve the GUI to provide a newsfragement.